Source/RxComponents/LMCPowerCombiner.cc

```diff
@@ -18,19 +18,19 @@ namespace locust
     PowerCombiner::PowerCombiner():
 			fnElementsPerStrip( 0 ),
 			fdampingFactors( 0 ),
-            fjunctionLoss( 1.0 ),
-            fpatchLoss( 0.6 ),
-            famplifierLoss( 0.66 ),
-            fendPatchLoss( 1.0 ),
-            fjunctionResistance( 0.3 ),
-            fnCavityProbes( 0 ),
+      fjunctionLoss( 1.0 ),
```

Comment:
I suggest that we maintain consistent indentation atleast within a file. I have had a lot of trouble reading through code in the past when not indented properly.

Reply:
Sorry. I think there was a conflict here between my editor's indentations and Penny's.
I changed a lot of indentation in ComplexFFT, which seems to have survived. I'll fix this one, too.